.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "src/lib/snarky"]
 	path = src/lib/snarky
-	url = git@github.com:o1-labs/snarky
+	url = ../../o1-labs/snarky
```

Comment:
will this require everyone have their snarky folder in the same location relative to their coda directory in this same fashion?

Reply:
This is a relative URL for cloning, not a path for the destination in the filesystem. I'm not sure how to parse your question.

Reply:
Ok, makes sense now. I thought this worked more like `npm`. I'm happy that it doesn't. Relevant quote from git documentation for others who are interested:

https://git-scm.com/docs/gitmodules#gitmodules-submoduleltnamegturl
> Defines a URL from which the submodule repository can be cloned. This may be either an absolute URL ready to be passed to git-clone[1] or (if it begins with ./ or ../) a location relative to the superproject’s origin repository.